tradebot/utils/rate_limiter.py

```
@@ -0,0 +1,389 @@
+    
+    def cleanup_old_records(self, keep_days: int = 30) -> int:
+        """
+        Clean up old quota tracking records.
+        
+        Args:
+            keep_days: Number of days to keep
+            
+        Returns:
+            Number of records deleted
+        """
+        cutoff_date = date.today().isoformat()
```

Ruff (0.11.9)

Local variable `cutoff_date` is assigned to but never used

Remove assignment to unused variable `cutoff_date`

(F841)